controllers/inverse_kinematics_pyikfast/inverse_kinematics_pyikfast.py

```diff
@@ -80,27 +80,74 @@
             sensors.append(sensor)
             sensor.enable(timeStep)
         except Exception as e:
-            print('Rotational Motor: ' + device.getName() +
-                  ' has no Position Sensor')
+            print("Rotational Motor: " + device.getName() + " has no Position Sensor")
+print("minPositions", minPositions)
+print("maxPositions", maxPositions)
+
 # --------------------------------------------------------------------
+# move the arm onto target using known solution
+target_position_joint_state = np.array([0, -np.pi / 2, np.pi / 2, 0, 0, 0]).astype(
+    np.double
+)
+def move_to_joint_state(motors, joint_state):
+    """Move robot endpoint to a known position that aligns it perfectly with the target"""
+    for i in range(len(joint_state)):
+        motors[i].setPosition(joint_state[i])
+# plase end-effector on target
+move_to_joint_state(motors, target_position_joint_state)
```

Comment:
what is the purpose of this? Does not seem necessary for the initialization phase of the controller